pkgs/applications/misc/passwordsafe/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, meson, ninja, pkg-config, gettext, fetchFromGitLab
+, python3, libhandy, libpwquality
+, wrapGAppsHook, gtk3, glib, gobject-introspection
+, desktop-file-utils, appstream-glib, gnome3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "passwordsafe";
+  version = "3.99.2";
```

Comment:
3.99.2 is a pre-release/dev version, we have a policy of packaging the latest stable release, or currently 3.32.0

Reply:
It seems that in latest version there's support implemented for newer versions of pykeepass
```
3.99.1
* Updated to pykeepass 3.2

3.99
* Implemented compatibility for pykeepass 3.1
```

It does appear this is a pre-release from looking at other releases (they follow the standard gnome owned pattern). But I'm confused by the implications from the version this will ship in `4.0`?

Reply:
This is also broken for nix in the 3.32.0 release, we would have to patch it, not sure if that has any bearing
https://gitlab.gnome.org/World/PasswordSafe/-/blob/3.32.0/meson.build#L31-40

Reply:
We could apply https://gitlab.gnome.org/World/PasswordSafe/-/commit/73b7aef155ce9ba597ebcadfb4345cbc966565e7 but if the pre-release version works, it might be more convenient to just use that.

Reply:
There are 3 other commits to that file after the 3.22.0 tag, so it cant be applied trivially, but anyways I tried with this patch, and it builds, but when I try to open a db it freezes and I have to kill -9 it
https://github.com/mvnetbiz/nixpkgs/blob/gnome-passwordsafe-3-22/pkgs/applications/misc/gnome-passwordsafe/meson.build.patch